### DISCUSSION TITLE
"Exit" Poll  ( Μιας Ωρας) Για την 1η ΟΣΣ

### DISCUSSION THREAD

**Discussion by Maurphal** [sdy60/syllabus#102] [Polls]
"Just one more turn" 

[4X](https://en.wikipedia.org/wiki/4X)

**Comment by iparthenios** [comment]
Συνάδελφοι ας το κάνουμε. Χρειάζεται πριν παραδώσουμε την εργασία...

**Comment by epidrome** [comment]
επειδή το ενδιαφέρον είναι σχετικά μικρό και εγώ μπορώ μόνο βραδυνές ώρες μήπως να γίνουν κάποιες απορίες ασύγχρονα στις συζητήσεις εδώ; γιατί μέχρι τώρα είναι ελάχιστες οι ασύγχρονες απορίες, ενώ και κάποιες που ζήτησα να γίνουν δεν έγιναν!
